### PR TITLE
Fix send all anyone can pay

### PR DESCRIPTION
**Pull request by classicalliu**

### CODE REVIEW

**Comment by Keith-CY** [comment]
CI failed

**Comment by classicalliu** [comment]
> CI failed

Passed now

File: packages/neuron-wallet/src/services/cells.ts [hunk 1]
```diff
@@ -526,6 +526,87 @@ export default class CellsService {
...
+
+    if (anyoneCanPayLockLiveCellEntities.length === 0) {
+      throw new CapacityNotEnough()
+    }
+
+    const inputs: Input[] = []
+    const inputOriginCells: LiveCell[] = []
+    let inputCapacities: bigint = BigInt(0)
+    let totalSize: number = baseSize
+    anyoneCanPayLockLiveCells.every(cell => {
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/services/cells.ts:566]: The returned value is fixed to `true` and not used so `Array#forEach` is more proper than `Array.every`

Comment (classicalliu) [reply packages/neuron-wallet/src/services/cells.ts:566]: ok

Comment (classicalliu) [reply packages/neuron-wallet/src/services/cells.ts]: updated

**Pull request merged**